test/e2e/csv_e2e_test.go

```diff
@@ -33,13 +32,22 @@ import (
 	"github.com/operator-framework/operator-lifecycle-manager/test/e2e/ctx"
 )
 
-var _ = Describe("CSV", func() {
-	It("create with unmet requirements mini kube version", func() {
+var _ = Describe("CSV represents a specific version of an Operator running on a cluster", func() {
+	var (
+		c   operatorclient.ClientInterface
+		crc versioned.Interface
+	)
+	BeforeEach(func() {
+
+		c = newKubeClient()
+		crc = newCRClient()
+	})
 
-		defer cleaner.NotifyTestComplete(true)
+	AfterEach(func() {
+		cleaner.NotifyTestComplete(true)
+	})
 
-		c := newKubeClient()
-		crc := newCRClient()
+	It("create with unmet requirements mini kube version", func() {
```

Comment:
Took me a minute to understand why there would be a test specific to minikube...

```suggestion
	It("create with unmet requirements MinKubeVersion", func() {
```

At a later date, these spec descriptions could use some revision -- it's not clear from reading most of them what behavior of OLM is being tested.